modules/logging/netwriter.go

```diff
@@ -134,6 +166,8 @@ func (reconn *redialerConn) Write(b []byte) (n int, err error) {
 	// we're the lucky first goroutine to re-dial the connection
 	conn2, err2 := reconn.dial()
 	if err2 != nil {
+		// logger socket seems to be offline; instead of discarding the log, dump it to stderr
+		os.Stderr.Write(b)
```

Comment:
Do we have some way of notifying the admin of the dial failure via stderr?
Something along the lines of:

```go
os.Stderr.Write([]byte("No connection to network log, message: "))
os.Stderr.Write(b)
```

Reply:
How often do you want to be notified? Every log message? Or one log message every 10 minutes, or...?

Reply:
I'd say every log message so that it immediately catches attention.
Since this is a temporary, probably important failure, IMO it would be appropriate.

Reply:
Would a single error message not "catch attention" though? Why would you need 100s of them (especially in hot paths).

Reply:
Was thinking that when there is only one error message, it would be hard to notice if there are many other log lines after that. Now that I think about it again, if someone was to look into the logs and see them being printed out, something can't be right as they should be sent to the network log instead. Should probably go with a single error message and see if it fits people's needs. For me it would be enough as I'd know the behavior.

Reply:
The idea behind my initial suggestion is just to let the user know that there is an issue (and what the issue is, in words) instead of just dumping the logs without such an error message. (`err` is returned here. maybe that's already enough?)